stackrox/acs-fleet-manager
# ROX-19718: Add flag for fleetmanager gitops

## Description

This PR exposes a new flag on fleet-manager to specify the path to the gitops configuration file. 
If the flag is not specified in the command-line args, it is assumed that no gitops configuration is used.

It also enables gitops by default on for development environments.   

## Checklist (Definition of Done)


## Test manual

1. Run as usual using `make deploy/dev`
2. Change the gitops configmap in dev/env/manifests/fleet-manager/04-gitops-config.yaml
3. Run `make deploy/dev/update-gitops-config` to update the configMap on k8s
4. Observe that the centrals are patched according to the config
